### PR TITLE
Fix BPMNDiagramMarshallerTest over-use of BPMN_EVALUATION test file.

### PR DESCRIPTION
**Pull request by ghost**
@romartin 
The other unit tests failed when you changed evaluation.bpmn because they also used this file (referenced using the constant BPMN_EVALUATION).

### CODE REVIEW

File: wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java [hunk 1]
```diff
@@ -346,7 +346,7 @@ public void testMarshallBoundaryEvents() throws Exception  {
 
     @Test
     public void testMarshallProcessVariables() throws Exception  {
-        Diagram<Graph, Settings> diagram = unmarshall(BPMN_EVALUATION);
+        Diagram<Graph, Settings> diagram = unmarshall(BPMN_PROCESSVARIABLES);
```
Comment (romartin) [inline wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java:349]: @jlindop agree on this one.

File: wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java [hunk 2]
```diff
@@ -308,9 +308,9 @@ public void testUmarshallBoundaryEvents() throws Exception  {
     
     @Test
     public void testUnmarshallSeveralDiagrams() throws Exception  {
-        Diagram<Graph, Settings> diagram1 = unmarshall(BPMN_EVALUATION);
-        assertDiagram( diagram1, 8 );
-        assertEquals( "Evaluation", diagram1.getSettings().getTitle() );
+        Diagram<Graph, Settings> diagram1 = unmarshall(BPMN_LANES);
```
Comment (romartin) [inline wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java:311]: This one does not makes sense, you're testing same lanes process twice on the unit test case. In fact, the idea of this test case is testing different marshall/unmarshall operations for different processes in a single run, to discard there are state-full component that can cause troubles. So I will let the code as it was :)

**Comment by romartin** [comment]
@jlindop Merged partially: 1f9af51c2193e32930503a1f6c2c470363216513
The evalutaion process tests are still failing, any ideas on those? Did u had a chance to give it a look?

**Comment by romartin** [comment]
Oh @jlindop I'm getting crazy with this...:( 

If adding a simple test like this one:

 @Test
    public void testMarshallEvaluationTwice() throws Exception  {
        Diagram<Graph, Settings> diagram = unmarshall(BPMN_EVALUATION);
        String result = tested.marshall(diagram);
        assertDiagram( result, 1, 7, 7);
        Diagram<Graph, Settings> diagram2 = unmarshall(BPMN_EVALUATION);
        String result2 = tested.marshall(diagram2);
        assertDiagram( result2, 1, 7, 7);
    }

It unmarshalls twice, and it's failing ONLY if using this serialized evaluation process -> https://gist.githubusercontent.com/romartin/bfa304987cb813c762b333fb36973721/raw/22c3e0667d45d5b9c1b51e4bf642b4060886c21d/gistfile1.txt

On the other hand, if using the current evalutaion.bpmn one, this same test is passing... does all this makes sense for you?

File: wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java (hunk 2 above)
Comment (ghost) [inline wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java:311]: @romartin 
Thanks - I should have read the name of the method more carefully!

**Comment by ghost** [comment]
@romartin I'll look at it these issues this morning.

File: wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java (hunk 2 above)
Comment (romartin) [inline wirez-sets/wirez-bpmn/wirez-bpmn-backend/src/test/java/org/wirez/bpmn/backend/service/diagram/BPMNDiagramMarshallerTest.java:311]: hehehe np :)

**Comment by romartin** [comment]
Closing

**Pull request closed without merging**